adapters/adapter.go

```diff
@@ -71,7 +71,7 @@ type Adapter interface {
 	OrderByRequest(r *http.Request) (values string, err error)
 	PaginateIfPossible(r *http.Request) (paginatedQuery string, err error)
 	ParseBatchInsertRequest(r *http.Request) (colsName string, colsValue string, values []interface{}, err error)
-	ParseInsertRequest(r *http.Request) (colsName string, colsValue string, values []interface{}, err error)
+	ParseInsertRequest(r *http.Request, table string, op string) (colsName string, colsValue string, values []interface{}, err error)
```

Comment:
we cant break that signature now, this would impact too many people

Reply:
Ok, please close this PR than, I will create new one with another proposed solution